Data-Structures/src/main/java/linkedList/LinkedList.java

```diff
@@ -171,4 +171,47 @@ public T getKthNodeFromTheEnd(int k) {
 
         return (T) currentNode.value;
     }
+
+    ////////////////////////////////////////////////////////////////////////////////////////////
+
+    // Write a function called mergeLists which takes two linked lists as arguments.
+    // Zip the two linked lists together into one so that the nodes alternate between the two lists
+    // and return a reference to the head of the zipped list. Try and keep additional space down to O(1).
+    // You have access to the Node class and all the properties on the Linked List class
+    // as well as the methods created in previous challenges.
+    public Node mergeLists(LinkedList linkedList1, LinkedList linkedList2) {

```

Comment:
This is good example of a method you could make static, since you are passing in two linked lists and then you won't have to call it on an instance. Like we do with Math.random()